advisories/2.1.0/BRSA-1j0O73qa.toml

```diff
@@ -0,0 +1,16 @@
+[advisory]
+id = "BRSA-1j0O73qa"
```

Comment:
One issue that didn't occur to me until seeing it in this context - we might want to restrict ourselves to lowercase letters as otherwise there could be filename collisions or general weirdness on case-insensitive filesystems.

Reply:
Fair enough; I can make that adjustment

Reply:
We should probably subsequently increase the length of the advisory IDs to reduce collision risk.

Reply:
Our chances of collision would increase from ~ 0.000000002% to ~ 0.0000002%. It is 2 orders of magnitude increase, but still extremely low. Even at 10,000 advisories over the lifetime of the project, the odds are ~0.00002% for ID collision. 

I think I'd prefer some sort of CI action on the advisories dir that would detect a collision for us

Reply:
Ah, I just remembered doing the math previously and was worried about invalidating our assumptions. But when we did the math, we had been talking about 4/5-digit IDs and not 8. I think you're right, that should be fine.